tools/clang/include/clang/SPIRV/SpirvInstruction.h

```diff
@@ -1968,7 +1977,7 @@ class SpirvDebugGlobalVariable : public SpirvDebugInstruction {
   SpirvVariable *var;
   // TODO: Replace this with an enum, when it is available in SPIRV-Headers
   uint32_t flags;
-  llvm::Optional<SpirvInstruction *> staticMemberDebugType;
+  llvm::Optional<SpirvInstruction *> staticMemberDebugDecl;
```

Comment:
I think this is never used right now. It'd be used for static class/struct members i believe. I'll open an issue so we don't forget about it.